### PR TITLE
ECO-5536 update dependencies

### PR DESCRIPTION
**Pull request by igorwojda**
- AGP
- gradlew
- dependencies

## Pull request checklist

- [ ] All tests pass. Demo project builds and runs.
- [ ] I have resolved any merge conflicts. Confirmation: ____

## What's in this pull request?
https://tokbox.atlassian.net/browse/ECO-5536
https://tokbox.atlassian.net/browse/ECO-5537

### CODE REVIEW

File: accelerator-core/build.gradle [hunk 1]
```diff
@@ -68,8 +68,8 @@ dependencies {
         maven { url 'http://tokbox.bintray.com/maven' }
     }
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'androidx.appcompat:appcompat:1.1.0'
-    implementation 'com.opentok.android:opentok-solutions-logging:+'
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'com.opentok.android:opentok-solutions-logging:1.0.2'
```
Comment (marinaserranomontes) [inline accelerator-core/build.gradle:72]: @igorwojda I think we could add 1.0.+ for logging.

**Review by marinaserranomontes — COMMENTED** [review]
LGTM

File: accelerator-core/build.gradle (hunk 1 above)
Comment (igorwojda) [reply accelerator-core/build.gradle:72]: We should avoid dynamic dependencies as they are considered bad practice and can cause some problems like broken builds or no easy way to build lib from a given day (eg. state of the repo from 2 months ago).

**Comment by igorwojda** [comment]
@ggoldens do we need to update `acceleratorCoreVersion` with each PR?

File: accelerator-core/build.gradle (hunk 1 above)
Comment (igorwojda) [reply accelerator-core/build.gradle:72]: https://blog.danlew.net/2015/09/09/dont-use-dynamic-versions-for-your-dependencies/

**Comment by marinaserranomontes** [comment]
@igorwojda yes, the acceleratorCoreVersion needs to be updated. It is used by the client logs

**Review by marinaserranomontes — APPROVED** [review]
LGTM

**Pull request merged**